resources/views/site/home.blade.php

```diff
@@ -34,7 +34,7 @@
                             @foreach($date_bookings as $b)
                                 <div class="bg-secondary px-2 py-1 mx-2 my-1 clearfix">
                                     <p class="my-0 float-left">{{ $b->callsign }}</p>
-                                    <p class="my-0 float-right">{{ $b->start_time_formatted }}-{{ $b->end_time_formatted }}</p>
+                                    <p class="my-0 float-right">{{ $b->start_time_formatted }}-{{ $b->end_time_formatted }}Z</p>
```

Comment:
This is causing the time to bump down to the next line. Will you please work on the margins/column width to ensure that this fits on a single line?
![image](https://github.com/ZTL-ARTCC/Current_Website/assets/21279582/30ad4e41-b6d5-46f5-9157-e5c0fb4de684)

Reply:
With the new changes, it does fix the stacking problem sometimes...

I originally made this stack on purpose for smaller screens so it wouldn't overflow (i.e. split the time onto 2 lines). Looking at the version with the old margins looks better IMO and doesn't stack on my monitor. 

I guess that being said, I think there are better ways to organize this to make it look better on all screens. I'm not sure if we should address that here or get some XD feedback (unless you all have suggestions). This was the best way I could think of, but I'm not the best designer, lol.

Reply:
@Evanston09 I agree with @iccowan... please undo the spacing changes and we'll merge it as you originally had. Addition additional width just compresses the event carousel too much. Sorry for the extra work.

Reply:
It has been reverted and no problem!